### DISCUSSION TITLE
Browser on RPI - Playback

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1454] [General]
Is it possible to set different "Playback - Settings" to the local browser ?

**Comment by ceejayemm** [comment]
What settings would you want ?

Chris

**Reply by hjfisch** [reply]
I build an Audioplayer with a small widescreen display .  There is no place for the covers underneath the song- and interpret name . If I switch off the covers in the playback-settings , they are also not shown on my Smartphone- and Laptop browser .

**Reply by hjfisch** [reply]
Or , if it is possible , it would be nice to show a small size cover beside the interpret- and songname .

**Comment by ceejayemm** [comment]
Does System / GPIO Devces / Character LCD or System / GPIO Devces / TFT 3.5 LCD help ?

Chris

**Reply by hjfisch** [reply]
Sadly not , it'a special Hdmi display as you can see in my former post.

**Reply by ceejayemm** [reply]
OK, out of ideas.

Chris

**Reply by hjfisch** [reply]
It was a pain to get it working.
